src/components/WelcomeModal/types.ts

```diff
@@ -0,0 +1,16 @@
+interface BaseCardType {
+  title?: string;
+  subtitle?: string;
+  description: string;
+  href?: string;
+}
+
+interface ImageCardType extends BaseCardType {
+  img?: string;
+}
+
+interface VideoCardType extends BaseCardType {
+  video?: string;
+}
```

Comment:
I feel like these props shouldn't be optional; if it's a `VideoCardType`, we should have a guarantee there's a `video`, and if it's a `ImageCardType`, we should have a guarantee there's an `img`